### ISSUE TITLE
Make documentation search more obvious

### ISSUE THREAD

**Issue by kimrutherford** [pombase/website#1955]
After @manulera announced the documentation search feature on pombelist we had the feedback below.  I think it makes sense to improve this discoverability of the search function.

See also: #929

From a PomBase user:

----------------------

Thanks for this news. I looked for this on the top webpage

https://www.pombase.org

and couldn’t find it. I suggest you list documentation in the drop down menu
under search

**Comment by manulera** [comment]
How do you envision this? I think it's nice that they are separate search boxes for genes/papers and the docs.

**Comment by ValWood** [comment]
I don't have any ideas how to make it clearer, but we should add a description of the documentation search here:
<img width="370" alt="Screenshot 2022-11-11 at 10 39 57" src="https://user-images.githubusercontent.com/7359272/201323540-d20892ab-c3f3-4a02-aa45-9e0f7b964c8e.png">

**Comment by kimrutherford** [comment]
> How do you envision this? 

I'm not sure.  :-)

Let's have a chat about it on Zoom sometime.

One possibility is to add "Search documentation" to the search menu, then link to the docs page, but with the search box highlighted:

![image](https://user-images.githubusercontent.com/90474/201323608-455b94d2-489d-49fb-b873-77ddb9386916.png)

**Comment by ValWood** [comment]
Actions.
add a new message to the Icon (i) to  
"or see the other options under under "search" menu.
add "Search documentation" link to the search menu

**Comment by kimrutherford** [comment]
I've added the extra line to the info icon text and added "Search documentation" to the search menu.

If you come to the documentation page via that menu, the search box will be highlighted and the cursor will be in the box, ready to type.

I've also changed the quick search box code so that the cursor will be in that box ("focused" in HTML terminology) when you open or re-load pombase.org.   Let me know if you see any problems with that change.
